README.md

```diff
@@ -19,11 +19,11 @@
 
 ### Installing via NPM.
 
-```$ npm i ytsearcher@1.2.3```
+```$ npm i ytsearcher@latest```
 
 ### Why ytsearcher?
 
-• Modern and trustworthy (promises, es6, up-to-date) with thousands of downloads.
+• Modern and trustworthy (typings(!), promises, es6, up-to-date) with thousands of downloads.
```

Comment:
Not really using es6 features but shrug

Reply:
no one asked

Reply:
@tbnritzdoge Do you have any complaints about this change? :D

Reply:
Doesn't seem to be any truth behind the comment. Thank goodness for OSS.